src/VirtualClient/VirtualClient.Monitors/FileUploadMonitor.cs

```diff
@@ -91,23 +91,20 @@ private async Task ProcessFileUploadsAsync(IBlobManager blobManager, EventContex
 
                 try
                 {
-                    await this.Logger.LogMessageAsync($"{this.TypeName}.ProcessFileUploads", relatedContext, async () =>
```

Comment:
Removed the logger here and added it in successive block to avoid the flooding up of traces with FileUploadMonitor log every few seconds.